pkgs/development/interpreters/php/7.4.nix

```diff
@@ -2,8 +2,8 @@
 
 let
   base = callPackage ./generic.nix (_args // {
-    version = "7.4.21";
-    sha256 = "0al2697d5hwq0f39rgncl1pwfxzzpc0afmr0fjvw5qjpww163v1n";
+    version = "7.4.23";
+    sha256 = "d1e094fe6e4f832e0a64be9c69464ba5d593fb216f914efa8bbb084e0a7a5727";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/3fbb55710d31b27c8fa5897e4e7714ea54ce06e5/pkgs/development/interpreters/php/generic.nix#L274-L277

I think both `nix-prefetch-url` and `nix-prefetch` will be okay in this case, the following two commands should produce the same result:

```plain
$ nix-prefetch-url --type sha256 "https://www.php.net/distributions/php-7.4.23.tar.bz2"
$ nix-shell -p nix-prefetch --run "nix-prefetch fetchurl --url https://www.php.net/distributions/php-7.4.23.tar.bz2"
```

Reply:
Ah nice! Thanks!!

Those commands are returning: `09spg854w25vigx4x4bg47xr7md59d36k75ych52x0sgdvz99q6i` which is different from what I have in this PR. How is this possible? How did I successfully built php 7.4.23 with a different sha256?

Reply:
We commonly use the base-32 representation (`09spg854w25vigx4x4bg47xr7md59d36k75ych52x0sgdvz99q6i`) rather than hexadecimal (`d1e094fe6e4f832e0a64be9c69464ba5d593fb216f914efa8bbb084e0a7a5727`), both is accepted.

Reply:
Thank you very much for the explanation!